src/App.svelte

```diff
@@ -1,6 +1,7 @@
 <script lang="ts">
   import "carbon-components-svelte/css/white.css";
 
+  import type { FeatureCollection, Polygon } from "geojson";
```

Comment:
For reference (because I couldn't find these anywhere in HTML docs nicely): https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/geojson/index.d.ts